README.md

```diff
@@ -1,15 +1,67 @@
 # KVIB - Kartverkets interaksjon- og designbibliotek
 
-[KVIB](https://kartverket.github.io/kvib/) er et prosjekt i regi av eiendomsportalen med mål om å samle designguidelines og kode for Kartverkets applikasjoner.
+[KVIB](https://kartverket.github.io/kvib/storybook) er monorepoet til Kartverkets designsystem. Kvib er et prosjekt i regi av eiendomsportalen med mål om å samle designguidelines og kode for Kartverkets applikasjoner.
```

Comment:
Tror ikke vi bør skrive noe om eiendom her. Det er riktignok eiendomsdivisjonen som tar mye av kostnaden, men det er IT som driver prosjektet, så tenker vi bare kan la være å nevne noe om det.

Reply:
> Jeg vet at mange utviklere liker godt å ha mest mulig kodenær dokumentasjon, og gjerne i readme-filer, men jeg blir allikevel automatisk litt skeptisk når jeg ser de samme instruksjonene gjentatt på flere steder. Det er så fort gjort at de kommer ut av sync og skaper forvirring. Er det mulig å løse dette ved å f.eks legge dokumentasjonen i storybook, men lenke til de riktige sidene fra readme? Evt andre veien?
> 
> Forøvrig synes jeg instruksjonene er veldig klare og gode! Har ikke testet å gjennomføre de enda, men det ser i hvert fall enkelt ut :)

Veldig godt poeng! Kanskje vi kan la det stå i readme i repoet og i storybook og lenke til storybooksiden fra npm?